src/hardware/targets.json

```diff
@@ -1364,6 +1364,14 @@
                 "platform": "esp8285",
                 "firmware": "Unified_ESP8285_2400_RX"
             },
+            "rp2pwm": {
+                "product_name": "RadioMaster RP2 2.4GHz RX PWM",
+                "lua_name": "RM RP2 PWM",
+                "layout_file": "Generic 2400 pwm.json",
```

Comment:
could probably use original file with `overlay` entry? just don't know how to disable/unset:
```
"serial_rx": 3,
    "serial_tx": 1,
```